qiskit/optimization/applications/ising/vehicle_routing.py

```diff
@@ -18,30 +18,35 @@
 checking its objective function value).
 """
 
+from typing import Tuple
 import numpy as np
 from qiskit.quantum_info import Pauli
 
+from qiskit.aqua.algorithms import MinimumEigensolverResult
 from qiskit.aqua.operators import WeightedPauliOperator
 
+# pylint: disable=invalid-name
 
-def get_vehiclerouting_matrices(instance, n, K):  # pylint: disable=invalid-name
+
+def get_vehiclerouting_matrices(instance: np.ndarray,
+                                n: int,
+                                K: int) -> Tuple[np.ndarray, np.ndarray, float]:
     """Constructs auxiliary matrices from a vehicle routing instance,
         which represent the encoding into a binary quadratic program.
         This is used in the construction of the qubit ops and computation
         of the solution cost.
 
     Args:
-        instance (numpy.ndarray) : a customers-to-customers distance matrix.
-        n (integer) : the number of customers.
-        K (integer) : the number of vehicles available.
+        instance: a customers-to-customers distance matrix.
+        n: the number of customers.
+        K: the number of vehicles available.
 
     Returns:
-        tuple(numpy.ndarray, numpy.ndarray, float):
+        Tuple[np.ndarray, np.ndarray, float]:
```

Comment:
I think this line can be dropped as this is already noted in the return type above.